src/test/resources/com/atomist/rug/test/gherkin/handler/event/PassingFeature1Steps.ts

```diff
@@ -7,6 +7,6 @@ When("a visionary leader enters", world => {
    world.registerHandler("ReturnsEmptyPlanEventHandler")
    world.sendEvent(new node.Commit)
 })
-Then("excitement ensues", (p,world) => {
+Then("excitement ensues", world => {
```

Comment:
What caught my eye was that when only one parameter is provided, it could be `world`. I've only ever seen the single parameter form have the one parameter be a `p: Project`.